modules/caddyhttp/requestbody/requestbody.go

```diff
@@ -94,8 +94,10 @@ type errorWrapper struct {
 
 func (ew errorWrapper) Read(p []byte) (n int, err error) {
 	n, err = ew.ReadCloser.Read(p)
-	if err != nil && err.Error() == "http: request body too large" {
-		err = caddyhttp.Error(http.StatusRequestEntityTooLarge, err)
+	if err != nil {
+		if _, ok := err.(*http.MaxBytesError); ok {
```

Comment:
```suggestion
	var mbe *http.MaxBytesError
	if errors.As(err, &mbe) {
```

Reply:
I'm sorry I didn’t see your comment earlier. I’ve made different changes. Specifically, I removed the redundant `nil` check and used the type assertion directly for `*HTTP.MaxBytesError` since the type assertion already guarantees that the error isn’t nil when ok is true.

That said, I really like this project and would love to be more involved. Would it be possible to connect on Slack? I’m eager to contribute more!

Reply:
Thanks. I still think `errors.As()` is recommended since the error value can be wrapped, thus failing a type assertion.

And yes! I'm running out the door but when I get back I'll send a Slack invite.